### ISSUE TITLE
Handler type error in Typescript

### ISSUE THREAD

**Issue by cjol** [fastify/fastify-funky#49]
### Prerequisites

- [X] I have written a descriptive issue title
- [X] I have searched existing issues to ensure the bug has not already been reported

### Fastify version

3.20.2

### Plugin version

1.0.2

### Node.js version

14.17.1

### Operating system

Linux

### Operating system version (i.e. 20.04, 11.3, 10)

20.04

### Description

Trying to integrate this in a typescript project gives Type errors on the simplest example I can think of.

### Steps to Reproduce

```ts
import Fastify from "fastify";
import { fastifyFunky } from "fastify-funky";

const fastify = Fastify();
fastify.register(fastifyFunky);
// this gives a type error:
fastify.get("/", (req, reply) => {
	return { right: { id: 1 } };
});
```

### Expected Behavior

The code compiles without type errors. I might expect to have to extend the Fastify interface type to do so.

**Comment by kibertoad** [comment]
@cjol Thank you for reporting! This is a limitation with fastify types, I've raised an issue there, hopefully we can address this upstream.

**Comment by kibertoad** [comment]
Released in 1.0.3
